backends/platform/ios7/ios7_video.mm

```diff
@@ -792,6 +793,24 @@ - (void)initSurface {
 
 	[self setViewTransformation];
 	[self updateMouseCursorScaling];
+    [self adjustViewFrameForSafeArea];
+}
+
+-(void)adjustViewFrameForSafeArea {
+    if ( @available(iOS 11, *) ) {
```

Comment:
`@available` is a pretty new addition to the clang compiler, and I suspect its use here will break for example the cross-compilation on our buildbot. Maybe using `if ([[UIApplication sharedApplication] keyWindow] respondsToSelector:@selector(safeAreaInsets)]` would work here instead?